### PR TITLE
Drawing case

### PR DESCRIPTION
**Pull request by Chroq**
## Description

Trello : https://trello.com/c/crypHsBj/1-2-as-norbert-i-want-to-start-a-new-game

In this PR, we draw the "grid" of hexagon. 

image

### CODE REVIEW

File: components/boards/Playboard.js [hunk 1]
```diff
@@ -1,10 +1,79 @@
...
+        <ButtomBoard
+          style={{
+            position: "absolute",
+            top: "-6.1%",
+            right: "-5%",
+            bottom: "-6.1%",
+            left: "-5%"
+          }}
+        ></ButtomBoard>
+        <div name="grid" style={{ position: "absolute", width: "100%", height: "100%" }}>
```
Comment (Kmaschta) [inline components/boards/Playboard.js]: You are mixing component style prop with style.jsx
Can you move a maximum of style into the style jsx ?
Positions can stay in style prop tho

File: components/boards/tools.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Hexagon from "./Hexagon";
+import ButtomBoard from "./BottomBoard";
+
+export function getBoardRatio(size) {
+  return (1 + (size - 1) * 0.75) / (size * 0.89 + (size * 0.89 - 0.89) / 2);
```
Comment (Kmaschta) [inline components/boards/tools.js]: Why 0.75 and 0.89 ? Can we name those values in a variables ?

File: components/boards/Playboard.js (hunk 1 above)
Comment (Chroq) [reply components/boards/Playboard.js]: Fixed, TY

File: components/boards/tools.js (hunk 2 above)
Comment (Chroq) [reply components/boards/tools.js]: Fixed, TY

File: components/boards/Playboard.js [hunk 3]
```diff
@@ -1,10 +1,75 @@
...
 function Playboard(props) {
   const size = props.size;
 
-  return <BottomBoard />;
+  const boardRatio = getBoardRatio(size);
+
+  const hexagonWidth = getHexagonWidth(size);
+  const hexagonHeight = getHexagonHeight(size);
+
+  const hexagons = () => {
```
[the remark this answers is not in the document]
Comment (Chroq) [reply components/boards/Playboard.js]: Fixed, TY

[the remark this answers is not in the document]
Comment (Chroq) [reply components/boards/Playboard.js]: Fixed, TY

File: components/boards/position.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+ * Hexagon width proportion
+ * An hexagon is not square. Approximatively, width is 0.89 unit of height.
+ */
+const hexagonWidthProportion = 0.89;
+
+/*
+ * Hexagon height three quarter
+ * Each hexagon is entangled in the two hexagons above. To represent this entanglement, we need to manage with the "three quarter" height.
+ */
+const hexagonHeightThreeQuarter = 0.75;
```
[the remark this answers is not in the document]
Comment (Chroq) [reply components/boards/position.js]: Done, Ty

**Pull request merged**